internal/driver/nodeserver_helpers.go

```diff
@@ -114,6 +114,18 @@ func validateNodeExpandVolumeRequest(req *csi.NodeExpandVolumeRequest) error {
 	return nil
 }
 
+// validateNodeUnpublishVolumeRequest validates the node unpublish volume request.
+// It checks the volume ID and target path in the provided request.
+func validateNodeUnpublishVolumeRequest(req *csi.NodeUnpublishVolumeRequest) error {
+	if req.GetVolumeId() == "" {
+		return errNoVolumeID
+	}
+	if req.GetTargetPath() == "" {
```

Comment:
At some places we are using req.GetRequestPath() and in some we are using req.TargetPath. Should we stick to one only?

Reply:
Good catch. Lets stick to using func() to get the values. I'll switch it